lib/api/client-commands/enablePerformanceMetrics.js

```diff
@@ -20,15 +20,15 @@ const {Logger} = require('../../utils');
  * @param {function} [callback] Optional callback function to be called when the command finishes.
  * @api protocol.cdp
  * @since 2.2.0
- * @see https://chromedevtools.github.io/devtools-protocol/tot/Performance/
+ * @see https://web.dev/metrics/
```

Comment:
this should be removed until we fix the tag reading in the website.

Reply:
Done.